test/integration/generator/main.go

```diff
@@ -0,0 +1,47 @@
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	"github.com/golang/glog"
+	"k8s.io/gengo/args"
+	"k8s.io/kube-openapi/pkg/generators"
+)
+
+const (
+	testdata_dir         = "./testdata/listtype"
+	output_package       = "generated"
+	output_base_filename = "openapi_generated"
+)
+
+func main() {
+	arguments := args.Default()
+	arguments.InputDirs = []string{testdata_dir}
+	arguments.OutputBase = "pkg"
```

Comment:
If we can make callers set the arguments instead of defaulting them, that will make the binary much more reusable.